csharp.test/TestWriterProperties.cs

```diff
@@ -41,8 +41,8 @@ public static void TestPropertiesBuilder()
                 .MaxRowGroupLength(789)
                 .Version(ParquetVersion.PARQUET_1_0)
                 .WriteBatchSize(666)
-                .DisableWritePageIndex()
                 .EnableWritePageIndex()
+                .DisableWritePageIndex()
```

Comment:
Is this change necessary?

Reply:
This was flipping the order of operations so that the end state is different to the new default, to check that these methods have an effect.